scripts/wasm-framework-ci.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Detect the architecture #
+if [[ $(arch) == "arm64" ]]; then
+    image="cosmwasm/rust-optimizer-arm64"
+    workspace_image="cosmwasm/workspace-optimizer-arm64"
+    abstract_image="abstractmoney/workspace-optimizer-arm64"
+else
+    image="cosmwasm/rust-optimizer"
+    workspace_image="cosmwasm/workspace-optimizer"
+    abstract_image="abstractmoney/workspace-optimizer"
+fi
+
+starting_dir=$(pwd)
+
+# see https://github.com/CosmWasm/cw-plus/blob/main/.circleci/config.yml
+echo "Wasming framework"
+cd ./framework
+
+# Remove for docker to successfuly copy
+rm packages/abstract-interface/state.json
+rm packages/abstract-interface/build.rs
+rm packages/abstract-interface/artifacts || true
+
+# Delete the current artifacts folder.
+# rm -rf ./artifacts
+
+# Create lock file if it does not exist
+if [ ! -f Cargo.lock ]; then
+    cargo generate-lockfile
+fi
+
+docker rm -v with_code || true
+
+# create a dummy container which will hold a volume with config
+docker create -v /code --name with_code alpine /bin/true
+# copy a config file into this volume
+docker cp Cargo.toml with_code:/code
+docker cp Cargo.lock with_code:/code
+# copy code into this volume
+docker cp ./contracts with_code:/code
+docker cp ./packages with_code:/code
+# Run the build
+docker run --volumes-from with_code ${abstract_image}:0.15.0
```

Comment:
For framework we should actually use the default image. 

The abstract image is for supporting the custom build features and name post-fixes but makes it harder to verify.